src/components/Students/StudentHeader.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+export default function StudentHeader(props) {
+
+const getStudentHeader = () => {
+    if( props.student.instrument == null ){
+        return (
+            <p>לא צוין כלי נגינה</p>
+        )
+    } else {
+        return (
+            <p>{props.student.instrument}</p>
+        )
+    }
+}
+
+
```

Comment:
Notice your spacings. I am not exactly sure what is the convention in React, but you should check it and follow a single convention. It makes the code more readable